docs/Latches/latches.md

```diff
@@ -18,4 +18,19 @@ There are two types of memory elements based on the type of triggering that is s
 Latches operate with enable signal, which is level sensitive. 
 Whereas, flip-flops are edge sensitive. 
 We will discuss about flip-flops in next module. 
-Now, let us discuss about SR Latch & D Latch one by one.
+Latches are basic storage elements that operate with signal levels (rather than signal transitions). Latches controlled by a clock transition are flip-flops. Latches are level-sensitive devices. Latches are useful for the design of the asynchronous sequential circuit.
```

Comment:
Good work, however, I think I have seen similar content somewhere. We need to be genuine about our work.
https://www.geeksforgeeks.org/latches-in-digital-logic/

Reply:
Yes, the content is taken from the link you mentioned. But I thought it would be fine to take the content from there as the link is already mentioned in the references section of the `Interactive Book` website. Please correct me here if I am wrong.

Reply:
I'll recommend not using the text as-in. 
You can use it for reference, no issues, but in order to add it to the site, it must be (at least) refactored to make it easy to understand/written in your own words/explanations added.

Reply:
I have made some changes, accordingly. Please review them as well. Thanks!